tests/lib/rules-preprocessor/gjs-gts-processor-test.js

```diff
@@ -270,7 +312,78 @@ describe('lint errors on the exact line as the <template> tag', () => {
       foo = 'bar';
       <template>
         <div>
-          some totally random, non-meaningful text 
+          some totally random, non-meaningful text
+        </div>
+      </template>
+    }
+    `;
+    const results = await eslint.lintText(code, { filePath: 'my-component.gjs' });
+
+    const resultErrors = results.flatMap((result) => result.messages);
+    expect(resultErrors).toHaveLength(1);
+    expect(resultErrors[0].message).toBe('Expected blank line between class members.');
+  });
+});
+
+describe('multiple tokens in same file', () => {
+  // TODO: this test still fails, needs to rework the token searching logic
+  // it('correctly maps duplicate tokens to the correct lines', async () => {
+  //   const eslint = initESLint();
+  //   const code = `
+  //     // comment one
+  //     // comment two
+  //     // comment three
+  //     const two = 2;
+
+  //     const three = <template> "bar" </template>
+  //   `;
+  //   const results = await eslint.lintText(code, { filePath: 'my-component.gjs' });
+
+  //   const resultErrors = results.flatMap((result) => result.messages);
+  //   expect(resultErrors).toHaveLength(2);
+
+  //   expect(resultErrors[0]).toStrictEqual({
```

Comment:
are these for a followup PR?

Reply:
yeah, let me do this: I'll just delete these tests, and open them up in a new PR to fix the token searching